distributions/distribution-resources/src/main/resources/conf/transform/readme.txt

```diff
@@ -0,0 +1,7 @@
+
+openHAB2 conf/transform folder
+
+Transformations like map or jsonpath can utilize configuration files with data definitions.
+These files have their specific file extensions and syntax definition.
+
+Check out the openHAB Wiki for more details: https://github.com/openhab/openhab/wiki/Transformations
```

Comment:
As these are add-ons as well, we should probably create a http://docs.openhab.org/addons/transformations.html page, which aggregates the documentation pages directly from the different transformations, what do you think?